resource/build-scripts/build-config-file.js

```diff
@@ -1,15 +1,29 @@
-'use strict';
-/**
- * External Dependencies
- */
-var base = require( '../../desktop-config/config-base.json' );
-var config = require( process.argv[2] );
+// TODO: evaluate if there is a more straightforward way to create config files e.g. how does calypso solve this?
+const CONFIG_SRC_DIR = '../../desktop-config/';
+const CONFIG_DEST = './desktop/config.json';
+
+const path = require( 'path' );
+const fs = require( 'fs' );
+const chalk = require( 'chalk' );
+
+const baseConfig = require( path.join( CONFIG_SRC_DIR, 'config-base.json' ) );
```

Comment:
it's kind of silly to use `path.join()` with this `CONFIG_SRC_DIR` constant when the constant is defined without `path.join()`

```js
const CONFIG_SRC_DIR = path.join( __dirname, '..', '..', 'desktop-config' );
const CONFIG_DEST = path.join( __dirname, 'desktop', 'config.json' );
```

I don't see inherent problems needing the `path.join()` but I would suggest it's better to go all-in or not at all so we don't introduce inconsistency